doc/notebooks/jupyter_integration.ipynb

```diff
@@ -0,0 +1,155 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Improved Integration of GRASS and Jupyter\n",
+    "\n",
+    "As part of Google Summer of Code 2021, we've been working to shorten and simplify the launch of GRASS in Jupyter and imporve the map displays. You can find out more abou the project and follow the progress on the [GRASS wiki page](https://trac.osgeo.org/grass/wiki/GSoC/2021/JupyterAndGRASS).\n",
+    "\n",
+    "This notebook is designed to run in binder and demonstrate the usage of `grass.jupyter`, the new module of Jupyter-specific functions for GRASS."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import subprocess\n",
+    "import sys\n",
+    "from IPython.display import Image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gisbase = subprocess.check_output([\"grass\", \"--config\", \"path\"], text=True).strip()\n",
+    "os.environ['GISBASE'] = gisbase\n",
+    "sys.path.append(os.path.join(gisbase, \"etc\",\"python\"))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import grass.script as gs\n",
+    "import grass.jupyter as gj"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "    This function initiates a GRASS session and sets GRASS\n",
+      "    environment variables.\n",
+      "\n",
+      "    Inputs:\n",
+      "        path - path to grass databases\n",
+      "        location - name of GRASS location\n",
+      "        mapset - name of mapset within location\n",
+      "    \n"
+     ]
+    }
+   ],
+   "source": [
+    "# Let's check our import by printing the __doc__ for init()\n",
+    "print(gj.init.__doc__)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Start GRASS Session\n",
+    "gj.init(\"../../data/grassdata\", \"nc_basic_spm_grass7\", \"user1\")\n",
+    "\n",
+    "# Set Display Settings\n",
+    "gj.display_settings(font=\"romans\", driver=\"cairo\") # Default font is \"sans\", let's change it to Romans just for fun!"
```

Comment:
I don't think `font="romans"` does what you expect. It uses the default font in GRASS display drivers which is not high quality. Perhaps `font="serif"` would give a good result. However, `grass.jupyter` is about reasonable defaults, so why to set anything at all in this basic example and not go with `gj.display_settings()`? This API and example will change, but this comment applies to the new API as well.